### DISCUSSION TITLE
Registration mess

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#17589] [Feature improvement]
Did snapshot of my production, sent it to my vps, changed config and cannot login. Stuck on registration which doesn't allow me go go anywhere. Registration here causing constant issues for users... may developer threating it with some sort of attention? Unless you like disappointing people of course

![image](https://github.com/user-attachments/assets/02d563a4-2bd6-4532-802f-19c008db57a8)

**Comment by justin025** [comment]
If it's just for development you could download ublock origin and mark the verification popup as an ad, but in production I recommended to just register.
